aws-janitor/resources/instance.go

```diff
@@ -54,8 +54,8 @@ func (Instances) MarkAndSweep(opts Options, set *Set) error {
 					Region:     opts.Region,
 					InstanceID: *inst.InstanceId,
 				}
-
-				if set.Mark(i) {
+				// Instances don't have a creation date, but launch time is better than nothing.
```

Comment:
Is that practically same thing? (I believe instances use LaunchXxx API call instead of CreateXxx)

Reply:
I think for our purposes, they'll generally be the same thing, yes. My reference is [this SO thread](https://stackoverflow.com/questions/18916135/determining-amazon-ec2-instance-creation-date-time), which notes that instances can potentially be restarted (without being recreated), which would reset this timestamp, but I doubt most tests will be doing that.